content/paths/tasks__post_tasks.yml

```diff
@@ -43,14 +43,15 @@ requestBody:
           action:
             type: string
             description: |-
-              The action the task assignee will be prompted to complete.
-
-              Currently only `review` is supported. This will provide the
-              assigned user the ability to approve or dismiss the
-              associated request.
+              The action the task assignee will be prompted to do. Must be
+              `review` for an approval task, or `complete` for a general
+              task. This will provide the assignee the ability to `approve` or
+              `reject` the associated file or request for approval tasks, and
+              the ability to `complete` general tasks.
```

Comment:
The second sentence here feels a bit heavy. Can we change it to two sentences instead, and maybe word it as "An approval task allows an assignee to `approve` or `reject` the associated file. A general task..."?